### ISSUE TITLE
[Q] The skipping canceled enqueue while training. 

### ISSUE THREAD

**Issue by chamcham115** [MarvinTeichmann/KittiSeg#61]
I attempt training using KITTI dataset. 
After 5 hours, It is canceled. 
Could you tell me the solution for this problem?

2017-06-05 21:12:34,341 INFO Step 11920/12000: loss = 0.22; lr = 1.00e-05; 0.215 sec (per Batch); 4.7 imgs/sec
2017-06-05 21:12:34,512 INFO       (raw)  Acc. : 0.99, xentropy: 0.02, weight_loss: 0.20
2017-06-05 21:12:34,512 INFO    (smooth)  Acc. : 0.99, xentropy: 0.01, weight_loss: 0.21
2017-06-05 21:12:45,605 INFO Step 11940/12000: loss = 0.22; lr = 1.00e-05; 0.222 sec (per Batch); 4.5 imgs/sec
2017-06-05 21:12:45,773 INFO       (raw)  Acc. : 1.00, xentropy: 0.01, weight_loss: 0.20
2017-06-05 21:12:45,773 INFO    (smooth)  Acc. : 0.99, xentropy: 0.01, weight_loss: 0.21
2017-06-05 21:12:56,879 INFO Step 11960/12000: loss = 0.23; lr = 1.00e-05; 0.222 sec (per Batch); 4.5 imgs/sec
2017-06-05 21:12:57,050 INFO       (raw)  Acc. : 0.99, xentropy: 0.02, weight_loss: 0.20
2017-06-05 21:12:57,050 INFO    (smooth)  Acc. : 0.99, xentropy: 0.01, weight_loss: 0.21
2017-06-05 21:13:08,226 INFO Step 11980/12000: loss = 0.22; lr = 1.00e-05; 0.224 sec (per Batch); 4.5 imgs/sec
2017-06-05 21:13:08,396 INFO       (raw)  Acc. : 0.99, xentropy: 0.01, weight_loss: 0.20
2017-06-05 21:13:08,397 INFO    (smooth)  Acc. : 0.99, xentropy: 0.01, weight_loss: 0.21
2017-06-05 21:13:18,961 INFO Running Evaluation Script.
2017-06-05 21:14:22.844011: I tensorflow/core/common_runtime/gpu/gpu_device.cc:977] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX TITAN X, pci bus id: 0000:01:00.0)
2017-06-05 21:14:22,984 INFO Evaluation Finished. All results will be saved to:
2017-06-05 21:14:22,984 INFO /home/chamcham/Dev/KittiSeg/hypes/../RUNS/KittiSeg_2017_06_05_16.43
2017-06-05 21:14:23,107 INFO Raw Results:
2017-06-05 21:14:23,107 INFO     [train] MaxF1 (raw)    :  98.1701 
2017-06-05 21:14:23,107 INFO     [train] BestThresh (raw)    :  48.2353 
2017-06-05 21:14:23,107 INFO     [train] Average Precision (raw)    :  92.5361 
2017-06-05 21:14:23,107 INFO     [val] MaxF1 (raw)    :  96.1826 
2017-06-05 21:14:23,107 INFO     [val] BestThresh (raw)    :  29.8039 
2017-06-05 21:14:23,107 INFO     [val] Average Precision (raw)    :  92.3907 
2017-06-05 21:14:23,107 INFO     Speed (msec) (raw)    :  152.7702 
2017-06-05 21:14:23,108 INFO     Speed (fps) (raw)    :  6.5458 
2017-06-05 21:14:23,108 INFO Smooth Results:
2017-06-05 21:14:23,108 INFO     [train] MaxF1 (smooth) :  98.0138 
2017-06-05 21:14:23,108 INFO     [train] BestThresh (smooth) :  46.4706 
2017-06-05 21:14:23,108 INFO     [train] Average Precision (smooth) :  92.5366 
2017-06-05 21:14:23,108 INFO     [val] MaxF1 (smooth) :  95.8499 
2017-06-05 21:14:23,108 INFO     [val] BestThresh (smooth) :  26.2745 
2017-06-05 21:14:23,108 INFO     [val] Average Precision (smooth) :  92.3395 
2017-06-05 21:14:23,108 INFO     Speed (msec) (smooth) :  152.7629 
2017-06-05 21:14:23,108 INFO     Speed (fps) (smooth) :  6.5461 
2017-06-05 21:14:38.006668: W tensorflow/core/kernels/queue_base.cc:294] _0_Queues/fifo_queue: Skipping cancelled enqueue attempt with queue not closed

**Comment by MarvinTeichmann** [comment]
The training is finished.
